lens-cube/src/main/java/org/apache/lens/cube/parse/StorageTableResolver.java

```diff
@@ -194,7 +182,7 @@ private void resolveDimStorageTablesAndPartitions(CubeQueryContext cubeql) throw
         boolean foundPart = false;
```

Comment:
Please leave a TODO that SkipStorageCause os deprecated and we need to handle that here.